virttest/utils_libvirt/libvirt_secret.py

```diff
@@ -3,12 +3,64 @@
 """
 
 import logging
+import re
+
+from aexpect import remote
+
+from avocado.core import exceptions
 
 from virttest import virsh
+from virttest.utils_test import libvirt
+from virttest.libvirt_xml import secret_xml
 
 LOG = logging.getLogger('avocado.' + __name__)
 
 
+def create_secret(sec_dict=None, remote_args=None):
+    """
+    Create a secret with 'virsh secret-define'
+
+    :param sec_dict: dict to create secret
+    :param remote_args: Parameters for remote host
+    :return: UUID of the secret
+    """
+
+    # Get dic from cfg file and give default dict value
```

Comment:
Sorry, I just found it. Is this comment necessary? If no, please delete it.  The sec_dict is a parameter of this function, people should not care about where it comes from, right?

Reply:
Yes ,it should be more simple ,I removed it